app/views/sections/index.html.erb

```diff
@@ -0,0 +1,100 @@
+<% require 'time' %>
```

Comment:
It's best to not use `require` in a view, if you need to require things it is usually done on an application wide level by putting it at the top of `config/application.rb`. But I believe that Rails already requires the `time` module for you.